<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoseRateSettingsDialog</class>
 <widget class="QWidget" name="DoseRateSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="dosimetrySettingsChambreLabel">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="dosimetrySettingsChambreLineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="creationLabel">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Date de cr√©ation</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="dateLineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="kqq0Label">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border:0px; background-color: none;</string>
      </property>
      <property name="text">
       <string>kqq0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="kqq0LineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="ndLabel">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border:0px; background-color: none;</string>
      </property>
      <property name="text">
       <string>Nd [cCgy/nC])</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="ndLineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="ndLabel_2">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border:0px; background-color: none;</string>
      </property>
      <property name="text">
       <string>Y offset [mm]</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="ndLabel_3">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border:0px; background-color: none;</string>
      </property>
      <property name="text">
       <string>Z offset [mm eau]</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="yoffsetLineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="zoffsetLineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
